src/filters.ts

```diff
@@ -7,3 +7,6 @@ export const byNoComments = (issue: IssueData): boolean => issue.comments === 0;
 
 export const isNotFromAuthor = ({ user }: IssueData, authors: string[]): boolean =>
   !authors.some((author) => author.toLowerCase() === user?.login.toLowerCase());
+
+export const byLabels = (issue: IssueData, labels: string[]): boolean =>
```

Comment:
Can we make it more descriptive?

```suggestion
export const byLabels = (issue: IssueData, requiredLabels: string[]): boolean =>
```